addons/gloot/ctrl_inventory_grid.gd

```diff
@@ -61,6 +61,7 @@ func _set_inventory(new_inventory: InventoryGrid) -> void:
 
 
 func _ready() -> void:
+	self.add_to_group("Inventory Grid")

```

Comment:
I think it is best to avoid using groups inside plugins.

The user might have some groups defined of his own and seeing some new ones in the list (seemingly out of nowhere) could be confusing. Or even worse, the user might accidentally use a group name that is already used by the plugin, breaking both, the plugin and the game logic.

Reply:
Maybe naming the group something specific like `gloot_ctrl_inventory_grid` or something like that might be okay, since nobody is going to make a group called that, and they would likely use spaces even if they did.

The only other sound solution i can think of would be using a global script that manages the inventory, but that's even worse. Perhaps iterating over every node in the tree, and checking if it has the method `_on_item_dropped()` or another method specific to the grid inventory? 

Besides, if the user is going to make a group for all grid nodes it changes nothing if it already exists. You can mention it in the documentation so they can use it without making their own. Upgrading to v2.0 already breaks some compatibility, so its better to make this fix sooner rather than later so the users don't have to remake their game twice.

Reply:
Even better, having a const function would be neat, so you could just call `Gloot.get_all_grid_invs()` and it would return an array you could iterate over. This could be useful for all inventories, not just grid. The very specific group name would likely never collide with a user-made group and it would not break old compatibility.

Reply:
I don't think global scripts are a bad idea. At least I think they are better than adding plugin-specific groups.
Documenting the group names could make things clearer, but I think that the less the user needs to know about the internals of a plugin (or any other lib), the better.

Besides, is tracking all instances of CtrlInventory* really necessary for "proper" item drag and drop? I would think that only tracking the dragged item would be sufficient.